logout.py

```diff
@@ -0,0 +1,32 @@
+# http://ptspts.blogspot.ca/2011/12/how-to-log-out-from-appengine-app-only.html
+
+import Cookie
+import os
+from google.appengine.api import users
+import webapp2
+import logging
+import authorizer
+
+class LogoutPage(webapp2.RequestHandler):
+  def get(self):
+    if os.environ.get('SERVER_SOFTWARE', '').startswith('Development/'):
```

Comment:
Logging out by setting cookies does not work on development environment (i.e. localhost).